src/core/Instrument.cpp

```diff
@@ -78,8 +82,96 @@ bool Instrument::isFromTrack( const Track * _track ) const
 	return( m_instrumentTrack == _track );
 }
 
+// helper function for Instrument::applyFadeIn
+int count_zero_crossings(sampleFrame *buf,fpp_t start, fpp_t frames)
```

Comment:
Can you make it a static function or a static member function and rename it to use camelCase?
Also, please add a whitespace after the first comma.

Reply:
Done!